db/migrate/20211109210319_create_bands.rb

```diff
@@ -0,0 +1,12 @@
+class CreateBands < ActiveRecord::Migration[6.1]
+  def change
+    create_table :bands do |t|
+      t.string :name, null: false
+      t.string :genre, null: false
+      t.text :biography
+      t.string :image, default: "https://swimg.com/wp-content/uploads/not-available.jpg"
```

Comment:
Let's come back to adding images when working on the image upload feature as there are better ways we can supply a default image